metron-deployment/roles/ambari_config/vars/single_node_vm.yml

```diff
@@ -102,7 +102,7 @@ configurations:
 
 required_configurations:
   - metron-env:
-      storm_rest_addr: "{{ groups.ambari_slave[0] }}:8744"
+      storm_rest_addr: "http://{{ groups.ambari_slave[0] }}:8744"
```

Comment:
@cestella Thoughts?  @dlyle65535 is right that it's the definition of a specific cluster.  I'd argue that it's the user's responsibility to update anything afterwards when they set up SSL (e.g. through Ambari).  All I was thinking of documenting was just to ensure that field gets updated, but I'm not sure how useful that is anyway.